src/batched_inference_executor.py

```diff
@@ -216,12 +214,13 @@ def run_infer_executor(self, num_reqs, i=0, is_warmup=False):
             completed += self.model_obj.infer()
             end_time = time.time()
 
-            total_time_arr.append(end_time - queued_time)
+            total_time = end_time - queued_time
+            total_time_arr.append(total_time)
             queueing_delay = start_time - queued_time
             queued_time_arr.append(queueing_delay)
 
             if is_percentile_calc:
-                self.opc.add_element(queueing_delay)
```

Comment:
Does it make sense to add the check `(not self.no_more_slo_grpc)` to the if here as well?